tests/pipelines/main.from_assemblies.nf.test

```diff
@@ -58,22 +58,22 @@ nextflow_pipeline {
                 def tokens = it.split("\t")
                 final_report_tmap[tokens[0]] = tokens[1]
             }
-            assert final_report_tsv[0] == "\tecoli_GCA_000947975\t"
+            assert final_report_tsv[0] == "\tecoli_GCA_000947975"
```

Comment:
Similar to the above comment I'm wondering if you could add an assert statement which verifies that the flattened report file exists in the IRIDA Next JSON output.

I'm wondering if you could make the listed changes near the following line in this file:

**below this line**
```groovy
 assert iridanext_global.findAll { it.path == "SummaryReport/final_report.tsv" }.size() == 1
```
**add this line**
```groovy
 assert iridanext_samples.ecoli_GCA_000947975.findAll { it.path == "SummaryReport/ecoli_GCA_000947975_flat_sample.json.gz" }.size() == 1
```

Reply:
fixed in: 10dc13b8f2c4dea5879e38ffde983753ea047967